app/src/androidTest/java/ch/epfl/sdp/blindly/main_screen/ProfilePageTest.kt

```diff
@@ -55,23 +55,25 @@ class ProfilePageTest {
     }
 
 
+    //TODO the tests seem really unstable need to look into this,
+    // sometimes they pass other time they failed because of "Error performing single click on",
+    // or "Wanted to match 1 intents. Actually matched 0 intents."
```

Comment:
This is very strange we sould look at it next week i think

Reply:
Yes I forgot to send the email with the stack trace to our assistants